recipe/meta.yaml

```diff
@@ -1,11 +1,17 @@
+{% set version = "0.9.0" %}
+
 package:
   name: tifffile
-  version: 0.7.0
+  version: {{ version }}
 
 source:
-  fn: tifffile-0.7.0.tar.gz
-  url: https://pypi.python.org/packages/source/t/tifffile/tifffile-0.7.0.tar.gz
-  md5: 38c1de9910a1e6b0faac0b84faf8ef72
+  fn: tifffile-{{ version }}.tar.gz
+  url: https://github.com/blink1073/tifffile/archive/v{{ version }}.tar.gz
+  sha256: 218f9c0640eff4d464c5619b1db57f049e92ff6cd2360d146332577f0ec2f312
+
+build:
+    number: 0
+    script: python setup.py install
```

Comment:
It looks like we are using `setuptools`, which I didn't realize.

Not that this is on you, but could we please update this with the canonical `--single-version-externally-managed --record=record.txt` and add `setuptools` explicitly to the build dependencies?

If you're too busy @ocefpaf, I can tack it on in a new PR later.